commodore/component/template.py

```diff
@@ -24,23 +35,23 @@ def cookiecutter_args(self) -> dict[str, str]:
             "add_golden": "y" if self.golden_tests else "n",
             "add_matrix": "y" if self.matrix_tests else "n",
             "copyright_holder": self.copyright_holder,
-            "copyright_year": self.today.strftime("%Y"),
+            "copyright_year": (
+                self.today.strftime("%Y")
+                if not self.copyright_year
+                else self.copyright_year
+            ),
             "github_owner": self.github_owner,
             "name": self.name,
             "slug": self.slug,
```

Comment:
Nitpick. We could push some of this logic to the parent class, like you did for `_initialize_from_cookiecutter_args`